common/src/main/scala/uk/ac/wellcome/sns/NotificationMessage.scala

```diff
@@ -0,0 +1,13 @@
+package uk.ac.wellcome.sns
+
+case class NotificationMessage(
```

Comment:
https://github.com/guardian/sns-event-stream/blob/master/app/sns/SubscriptionConfirmationMessage.scala#L42

@robertberry 👏

Reply:
This is a better equivalent of [this](https://github.com/wellcometrust/platform/blob/master/common/src/main/scala/uk/ac/wellcome/models/aws/SQSMessage.scala). Can we add a TODO there to remove it?